examples/with-jest/index.test.js

```diff
@@ -0,0 +1,19 @@
+import {matcher, serializer} from 'jest-glamor-react'
+import React from 'react'
+import {shallow, render, mount} from 'enzyme'
+import {Wrapper, Title} from './index'
+
+expect.addSnapshotSerializer(serializer)
```

Comment:
How do I get `expect` inside setupFile? 
`ReferenceError: expect is not defined` is what I get.